### ISSUE TITLE
OME-TIFF metadata IO support in the TIFF ImageIO

### ISSUE THREAD

**Issue by thewtex** [InsightSoftwareConsortium/ITK#3427]
### Description

Add OME-TIFF metadata IO to the ITK TIFF ImageIO. OME-TIFF is the most standard TIFF metadata format in adopted by microscopy community.

### Expected behavior

[OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.2.2/ome-tiff/) metadata is read and written by ITK's TIFF ImageIO. This in includes spatial metadata and additional metadata in the Image MetaDataDictionary.

### Actual behavior

This metadata is not used.

### Additional Information

Spatial metadata can be stored in `PhysicalSizeX`, `PhysicalSizeY`, and the per-`Plane`: `PositionX`, `PositionY`, `PositionZ`.

**Comment by blowekamp** [comment]
As I recall the OME-TIFF meta data is most frequently XML encoded in a TIFF TAG. Is this proposal to place that XML into the ITK's MetaData dictionary or further parse the meta-data information into ITK's intrinsic metadata (spacing, origin etc). Both would be useful, but looking for clarification for the goals of this issue.

**Comment by thewtex** [comment]
Yes, we should read / write the OME-XML tag. The spatial metadata (items noted in *Additional Information*) should go into `itk::Image` `Origin` and `Spacing`. The rest into `MetaDataDictionary`.

**Comment by zhusihan-python** [comment]
hello, is there any updates about the ome tiff support in itk 5.3.0

**Comment by dzenanz** [comment]
I don't think there has been any progress on this front.

**Comment by zhusihan-python** [comment]
> I don't think there has been any progress on this front.

hi sir dzenanz, when is the ZarrNGFF will be ready, is it better than the ome tiff?

**Comment by dzenanz** [comment]
OME-TIFF support is not even started. ZarrNGFF is in early stages. Which one will be better? Probably the one which receives more attention. That seems to be ZarrNGFF.
